components/datadog/updater/install_script.sh

```diff
@@ -50,12 +50,12 @@ if [ "${ARCH}" = "aarch64" ]; then
 fi
 
 apt_url="apttesting.datad0g.com"
-apt_repo_version="${DD_PIPELINE_ID}-u7-${ARCH} 7"
+apt_repo_version="${DD_PIPELINE_ID}-i7-${ARCH} 7"
 apt_usr_share_keyring="/usr/share/keyrings/datadog-archive-keyring.gpg"
 apt_trusted_d_keyring="/etc/apt/trusted.gpg.d/datadog-archive-keyring.gpg"
 
 yum_url="yumtesting.datad0g.com/testing"
-yum_repo_version="${DD_PIPELINE_ID}-u7/7"
+yum_repo_version="${DD_PIPELINE_ID}-i7/7"
```

Comment:
you'll need to carefully add this change on the same PR that updates the versions or updater-e2e tests will break